### PR TITLE
chore(deps): update deps related to bintray

### PR DESCRIPTION
**Pull request by nikpachoo**
I replaced the bintray links with the current ones that I got from the [master branch](https://github.com/AlexanderPrendota/kotlin-compiler-server/blob/master/build.gradle.kts#L56).

### CODE REVIEW

**Comment by AlexanderPrendota** [comment]
@woainikk @anastasiiaSpaseeva Folks, please validate deps

File: build.gradle.kts [hunk 1]
```diff
@@ -95,7 +96,7 @@ dependencies {
     implementation("org.jetbrains.kotlin:kotlin-compiler:$kotlinVersion")
     implementation("org.jetbrains.kotlin:kotlin-script-runtime:$kotlinVersion")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-js:$kotlinVersion")
-    implementation("org.jetbrains.kotlin:kotlin-compiler-for-ide:$kotlinVersion")
+    implementation("org.jetbrains.kotlin:kotlin-compiler-for-ide:$kotlinVersion-release-310")
```
Comment (AlexanderPrendota) [inline build.gradle.kts:99]: I think it;s a bad idea to pass raw string "-release-310" to deps. We always update dependencies via gradle.properties file

File: common/build.gradle.kts [hunk 2]
```diff
@@ -7,5 +7,10 @@ plugins {
 
 dependencies {
     implementation("org.jetbrains.kotlin:kotlin-compiler:$kotlinVersion")
+    implementation("org.jetbrains.kotlin:kotlin-compiler-for-ide") {
+        version {
+            strictly( "$kotlinVersion-release-310")
```
Comment (AlexanderPrendota) [inline common/build.gradle.kts:12]: why do u need such changes?

**Review by AlexanderPrendota — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (woainikk) [reply build.gradle.kts:99]: there should be used $kotlinIdeVersion

File: common/build.gradle.kts (hunk 2 above)
Comment (nikpachoo) [reply common/build.gradle.kts:12]: The problem is that the dependency `org.jetbrains.kotlin:common:202-$kotlinIdeVersion-IJ8194.7` refers to "org.jetbrains.kotlin:kotlin-compiler-for-ide-1.4.30" which cannot be resolved cause there is no such version ([mvnrepository](https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-compiler-for-ide)).

Comment (woainikk) [reply common/build.gradle.kts:12]: why kotlin 1.4.30 version used, not the new one (1.5.30)?

File: build.gradle.kts (hunk 1 above)
Comment (AlexanderPrendota) [reply build.gradle.kts:99]: riiiiiight

**Comment by AlexanderPrendota** [comment]
ah exactly, the PR related to kotlin 1.4.30

File: common/build.gradle.kts (hunk 2 above)
Comment (AlexanderPrendota) [reply common/build.gradle.kts:12]: because the pr into 1430)

File: build.gradle.kts (hunk 1 above)
Comment (nikpachoo) [reply build.gradle.kts:99]: I've tried to use $kotlinIdeVersion, but it didn't work. I got an error:
```
Could not find org.jetbrains.kotlin:kotlin-compiler-for-ide:1.4.30-release.
```

Comment (AlexanderPrendota) [reply build.gradle.kts:99]: kotlinVersion=1.4.30-release-310 ?

**Review by AlexanderPrendota — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (nikpachoo) [reply build.gradle.kts:99]: > kotlinVersion=1.4.30-release-310
nope, because the other dependencies that use this variable stop working.

**Review by woainikk — APPROVED** [review]

**Pull request merged**